multi-node/aws/pkg/config/config_test.go

```diff
@@ -34,6 +34,11 @@ vpcId: vpc-xxxxx
 routeTableId: rtb-xxxxxx
 `, `
 vpcId: vpc-xxxxx
+`, `
+createRecordSet: false
+hostedZone: ""
+`, `
+recordSetTTL: 400
```

Comment:
should this case maybe be invalid?  a non-default `recordSetTTL` with `createRecordSet=false` is non-sensical?